ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -84,10 +91,29 @@ export class ImportCommand extends IronfishCommand {
 
     process.stdin.off('data', onData)
 
-    return JSONUtils.parse<AccountImport>(data)
+    try {
+      const decodedWords = bech32m.bech32m.decode(data).words
+      const decodedBytes = bech32m.bech32m.fromWords(decodedWords)
+      return JSONUtils.parse<AccountImport>(decodedBytes.toString())
+    } catch (e) {
+      return JSONUtils.parse<AccountImport>(data)
+    }
   }
 
   async importTTY(): Promise<AccountImport> {
+    const bech32 = await CliUx.ux.prompt(
+      "Enter the bech32 encoded account (string beginning with 'ironfishprivatekey'), or hit enter to begin legacy hex spending key workflow",
```

Comment:
The prefix `ironfishprivatekey` is a misnomer, this is actually an an entire account that is encoded in json.